crates/router/src/core/payments/flows.rs

```diff
@@ -2759,3 +2763,115 @@ default_imp_for_post_session_tokens!(
     connector::Wellsfargopayout,
     connector::Wise
 );
+
+/// Determines whether a capture API call should be made for a payment attempt
+/// This function evaluates whether an authorized payment should proceed with a capture API call
+/// based on various payment parameters. It's primarily used in two-step (auth + capture) payment flows for CaptureMethod SequentialAutomatic
+///
+pub fn should_initiate_capture_flow(
+    connector_name: &router_types::Connector,
+    off_session: Option<bool>,
+    customer_acceptance: Option<CustomerAcceptance>,
+    capture_method: Option<api_enums::CaptureMethod>,
+    setup_future_usage: Option<api_enums::FutureUsage>,
+    status: common_enums::AttemptStatus,
+) -> bool {
+    match status {
+        common_enums::AttemptStatus::Authorized => {
+            if let Some(api_enums::CaptureMethod::SequentialAutomatic) = capture_method {
+                match connector_name {
+                    router_types::Connector::Paybox => {
+                        // Check CIT conditions for Paybox
+                        (off_session == Some(true)
```

Comment:
is off_session == true a condition for CIT?

Reply:
removed it